packages/web_ui/src/components/InstanceStatusTag.jsx

```diff
@@ -0,0 +1,21 @@
+import { Tag } from "antd";
+import React from "react";
+
+const statusColors = {
+	unknown: "#d46b08",
+	unassigned: "#d4380d",
+	stopped: "#cf1322",
+	starting: "#d4b106",
+	running: "#52c41a",
+	stopping: "#d48806",
```

Comment:
How about something like this for stopping - an orange color: #ff9f29

Reply:
For the unknown may be some other colour would fit, something that stands out and not used other for statuses. For example: #ff00f7
And for unnasigned, maybe something gray?